modules/weko-workflow/weko_workflow/api.py

```diff
@@ -1633,6 +1633,25 @@ def get_activity_by_workflow_id(self, workflow_id):
             current_app.logger.error(ex)
             return None
 
+    def update_title(self, activity_id, title):
+        """
+        Update title to activity.
+
+        :param activity_id:
+        :param title:
+        :return:
+        """
+        try:
+            with db.session.begin_nested():
+                activity = self.get_activity_detail(activity_id)
+                if activity:
+                    activity.title = title
+                    db.session.add(activity)
```

Comment:
You should use `db.session.merge(activity)` instead of `db.session.add(activity)`